packages/fxa-admin-server/src/subscriptions/appstore.service.ts

```diff
@@ -3,22 +3,25 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Firestore } from '@google-cloud/firestore';
-import { Inject, Injectable } from '@nestjs/common';
+import { Inject, Injectable, LoggerService } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
-import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
 import { AppStoreHelper } from 'fxa-shared/payments/iap/apple-app-store/app-store-helper';
 import { PurchaseManager } from 'fxa-shared/payments/iap/apple-app-store/purchase-manager';
 import { AppConfig } from '../config';
 import { FirestoreService } from '../backend/firestore.service';
+import { LOGGER_PROVIDER } from '@fxa/shared/log';
 
 /**
  * Extends AppStoreHelper to be service like
  */
 @Injectable()
 export class AppStoreHelperService extends AppStoreHelper {
-  constructor(configService: ConfigService, logger: MozLoggerService) {
+  constructor(
+    configService: ConfigService,
+    @Inject(LOGGER_PROVIDER) logger: LoggerService
```

Comment:
Could the typing here be changed back to `MozLoggerService`? This would also get rid of the cast to `any` below.

Also makes it easier to spot where refactoring is required in future when switching to `LoggerSerivce` and away from `mozlog`.

Reply:
Yep, will let it remain as that.